consensus/hotstuff_leader.go

```diff
@@ -18,48 +21,59 @@ var (
 	}
 )
 
-type HotstuffLeaderMessageHandler struct{}
-
 /*** Prepare Step ***/
 
 func (handler *HotstuffLeaderMessageHandler) HandleNewRoundMessage(m *ConsensusModule, msg *typesCons.HotstuffMessage) {
 	handler.emitTelemetryEvent(m, msg)
+	defer m.paceMaker.RestartTimer()
```

Comment:
I feel a bit battled about inverting the order of these two lines (and below as well).

Reply:
Update all-the-things!